.gitignore

```diff
@@ -11,9 +11,9 @@
 # Autogenerated VS/MD/Consulo solution and project files
 ExportedObj/
 .consulo/
-*.csproj
+**/*.csproj
```

Comment:
These files are still being tracked. 

`git rm --cached <file_path>` will remove them from the repo and leave the file on the local machine